Pop-Grupa-A-Frontend/src/commons/ApiLinks.js

```diff
@@ -2,10 +2,12 @@ const baseUrl = "http://localhost:5000/"
 
 const taskUrl = `${baseUrl}task/` // plus app
 const APP_URL = `${baseUrl}application/`
+const NODE_URL = `${baseUrl}nodes/`
```

Comment:
Jeśli przeniesiecie metody do clusterService to nie będzie NODE_URL potrzebne

Reply:
tak samo jak wyżej napisałem, można śmiało usunąć